core-bundle/contao/dca/tl_content.php

```diff
@@ -1005,8 +1005,16 @@ public function addCteType($arrRow)
 			$class .= ' empty';
 		}
 
+		$dragHandle = '';
+
+		if (!Input::get('act'))
```

Comment:
Shouldn't we check `$security->isGranted(ContaoCorePermissions::DC_PREFIX . 'tl_content', new UpdateAction('tl_content', $arrRow))` here?

Reply:
We're not checking permissions here, we are checking if this is the list view.

Reply:
But why add the drag handle if we already know that the user does not have permission to use it?

Reply:
Ah you're right, I thought you mean instead of the `Input::get('act')`. Fixed in 4fe3dd9a5f9c77118fbb44679e9c5a8cea1ce47f